pkg/ebpf/c/tracer-ebpf.c

```diff
@@ -547,7 +510,13 @@ int kprobe__tcp_sendmsg(struct pt_regs* ctx) {
     }
     log_debug("kprobe/tcp_sendmsg: pid_tgid: %d, size: %d\n", pid_tgid, size);
 
-    return handle_message(sk, status, pid_tgid, CONN_TYPE_TCP, size, 0);
+    conn_tuple_t t = {};
+    if (!read_conn_tuple(&t, status, sk, pid_tgid, CONN_TYPE_TCP)) {
+        return 0;
+    }
+
+    handle_tcp_stats(&t, status, sk);
```

Comment:
*if* this turns out to have a perf penalty we could also sample this function call, and only update the RTT value like 10% of the time. this wouldn't work in case of retransmits though.

Reply:
`bpf_get_prandom_u32()` can help us here